AppCenterData/AppCenterDataTests/MSDBDocumentStoreTests.m

```diff
@@ -73,7 +73,7 @@ - (void)testReadAppDocumentFromLocalDatabase {
   // If
   NSString *documentId = @"12829";
   NSString *eTag = @"398";
-  NSString *jsonString = @"{ \"document\": {\"key\": \"value\"}}";
```

Comment:
Is a specific reason to change the test string here? :)

Reply:
Yes, now we do not store the root json string in storage, just the json of the document property